src/connection.cpp

```diff
@@ -278,6 +285,21 @@ void HttpConnectionManager::ev_handler_http(mg_connection *client, int ev, void
 
             mg_str& body = hm->body;
             prms.input.load(body.p, body.len);
+
+            for(int i = 0; i < MG_MAX_HTTP_HEADERS; ++i)
+            {
+                const mg_str& h_n = hm->header_names[i];
+                const mg_str& h_v = hm->header_values[i];
+
+                if(h_n.p == nullptr)
+                    break;
+
+                prms.input.headers.push_back({std::string(h_n.p, h_n.len), std::string(h_v.p, h_v.len)});
+            }
+
+            prms.input.remote_host = remote_address_host_str;
+            prms.input.remote_port = remote_port;
```

Comment:
It is not required from now. I have fixed it. Sorry for this.

Reply:
It was added as a pair for host. I agree in most cases this is useless. However,  it may be used to identify connection using firewall.

Reply:
remote_host has been removed